onnxruntime/test/fuzzing/src/OnnxPrediction.cpp

```diff
@@ -2,144 +2,125 @@
 // Licensed under the MIT License.
 
 #include "OnnxPrediction.h"
+#include "onnxruntime_session_options_config_keys.h"
+
+#include <vector>
 
 // Uses the onnxruntime to load the model
 // into a session.
 //
-OnnxPrediction::OnnxPrediction(std::wstring& onnx_model_file)
-:
-rawModel{nullptr},
-ptrSession{nullptr},
-session{ env, onnx_model_file.c_str(), emptySessionOption},
-inputNames{session.GetInputCount()},
-outputNames{session.GetOutputCount()}
-{
+OnnxPrediction::OnnxPrediction(std::wstring& onnx_model_file, Ort::Env& env)
+    : raw_model{nullptr},
+      ptr_session{std::make_unique<Ort::Session>(env, onnx_model_file.c_str(), empty_session_option)},
+      input_names(ptr_session->GetInputCount()),
+      output_names(ptr_session->GetOutputCount()) {
   init();
 }
 
 // Uses the onnx to seri
 //
-OnnxPrediction::OnnxPrediction(onnx::ModelProto& onnx_model)
-:
-session{nullptr}
-{
-  rawModel = std::shared_ptr<void>{alloc.Alloc(onnx_model.ByteSizeLong()),   
-  [this](void * ptr)
-      { 
-      this->GetAllocator().Free(ptr); 
-      }
-  };
+OnnxPrediction::OnnxPrediction(onnx::ModelProto& onnx_model, Ort::Env& env) {
+  raw_model = std::shared_ptr<void>(alloc.Alloc(onnx_model.ByteSizeLong()), [this](void* ptr) {
+    this->GetAllocator().Free(ptr);
+  });
 
```

Comment:
nit: Add TODO saying this doesn't need to use shared_ptr or alloc (can be done with 'new' or a std::vector based buffer.